free-3.md

```diff
@@ -45,6 +59,9 @@ aren't and conjecture why not. If so, describe them and provide links. How well
 do they address the need? Are there any particularly admirable qualities of the
 language? Are there parts of the language you think could be improved?_
 
+I found [this website](http://www.howmanysyllables.com/), but it doesn't check if words are even real and only tells you how many syllables you have. For things like a sonnet, where lines have 10 syllables at most, this doesn't seem all that useful, though it does tell you syllables per line. I think there is still space for some sort of "sonnet checker" or general poem checker as this website doesn't do quite enough for me.
+
+I found a few other syllable counters, but they all kind of do the same thing as the above website (and probably a bit worse).
```

Comment:
Rhyming dictionaries are a thing too! But I haven't seen anything quite like this either.